shipanaro/models.py

```diff
@@ -47,33 +53,35 @@ class Membership(models.Model):
     contact_id = models.CharField(max_length=9)
     date_left = models.DateField(null=True, blank=True)
     drop_out = models.BooleanField(default=False)
+    # membership applications pending acceptance
+    date_accepted = models.DateField(null=True, blank=True)
+    activated = models.BooleanField(default=False)
```

Comment:
Once new affiliates can apply via here, we can use these fields to manage "application approval"